features/steps/project/source/browse_files.rb

```diff
@@ -35,7 +35,7 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
   step 'I should see its new content' do
     expect(page).to have_content new_gitignore_content
   end
-
+ 
```

Comment:
@tsigo , thanks, I need to do that, just in my local vim editor, I did not find whitespace there, sorry.